### ISSUE TITLE
Teams App Update Permission Requirements?

### ISSUE THREAD

**Issue by erik365online** [MicrosoftDocs/msteams-docs#2863]
Could you please clarify this page more in regards on required permissions for uploading a app and especially regarding updating an app in the TAC\teams app area.  **What permissions are required to update an existing update with a new version?** 
I'm on a case with Microsoft Premier Support and they've referenced to this page but to me this is insufficient.
Users and persons with the Teams Admin Role can upload new custom apps but the update of an existing app with a new version fails we'd like to exclude a permission issue and therefore need proper information about required permissions. Thank you.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 54780e5a-ca3a-5277-dc84-37f85d9ea1cb
* Version Independent ID: 5f89865b-a9fd-ada5-72d2-6375357bba8c
* Content: [Upload your custom app - Teams](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/deploy-and-publish/apps-upload)
* Content Source: [msteams-platform/concepts/deploy-and-publish/apps-upload.md](https://github.com/MicrosoftDocs/msteams-docs/blob/master/msteams-platform/concepts/deploy-and-publish/apps-upload.md)
* Product: **msteams**
* GitHub Login: @laujan
* Microsoft Alias: **lajanuar**

**Comment by RamaMohanaChary** [comment]
Hi @erik365online,

Could you please try by uninstall and install your application. Once you install the application again, if you still face any issue please provide us the repro steps and screenshots.

**Comment by erik365online** [comment]
Hi @RamaMohanaChary, 
Thank you. The app is already used and pinnend in production. Deleting it would cause that users in their Teams client apps have to re-pin other previously pinned apps because the app is distributed and pinned using a Teams Setup Policy.

Today, I'll check with the Premier Support and collect some logs with Fiddler. We did already a PSR so that the support can see what I was doing and how I was updating the app. Adding new custom apps is working fine but deleting a custom app with an older version and adding the custom app with the new version as I would do for any new custom app is not how it should work in my opinion. 

I'll proceed with the Premier Support and come back here as soon as I've got something new.
Best Regards Erik
